### ISSUE TITLE
[Screen Reader - LAPS-Domain Controller - Custom Setup] Screen reader does not read the level of tree items which are present in custom setup tree hierarchy

### ISSUE THREAD

**Issue by sonimishra** [wixtoolset/issues#5813]
## Bugs

* Which version of WiX are you building with?
  3.7.4522.0 

People who depend on screen reader  will not get complete information regarding tree hierarchy if screen reader does not read the level of tree items.

**Repro Steps:**
1-Go to windows icon and search for remote desktop.
2-Go to the name field and provide the IP address(domain controller) to connect to the domain controller VM.
3-Provide valid username and password for successful connection.
4-Go to LAPS named folder in the desktop and select the executable file:- LAPS 64 bit. Right click on this file and select Install button.
5-Navigate to the Run button appearing on the window and select it.
6-Navigate to various controls lying on the LAPS setup window opened and select Next button.
7-Navigate to various controls lying on the End user License Agreement window opened and select Next button.
8-Navigate to various controls lying on the Custom setup window opened. Navigate to the tree items present on tree hierarchy.
10. Verify that screen reader does not read the level of tree items .

**Actual:** Screen reader does not read the level of tree items which are present in custom setup tree hierarchy. Screen reader read eg: management tools 2 of 2 but it does not read the level like 'level 1' 

**Expected:** Screen reader should read the level of tree items which are present in custom setup tree hierarchy. Screen reader should read like eg: management tools level 1 , 2 of 2 " because this tree item has its sub Child.
